### DISCUSSION TITLE
How to modify translated files

### DISCUSSION THREAD

**Discussion by foxiyu** [dail8859/NotepadNext#645] [General]
It seems that there is no Chinese translation in the options .How to modify translated files.

When can the dark mode or custom theme be released

**Comment by foxiyu** [comment]
Do I need to remove type="unfinished" after translating Chinese
![image](https://github.com/user-attachments/assets/6d580cf4-b3aa-4bea-a055-5ce3eef20627)

**Reply by dail8859** [reply]
That is the file that has all the translations but it needs to be edited with the Qt Linguist tool. You can see more details here:

https://github.com/dail8859/NotepadNext/blob/master/doc/Translation.md#translation-process-for-translators

**Reply by foxiyu** [reply]
Okay, thank you very much

**Comment by MeiyongYu** [comment]
I am not use dark often, i donot understard why people like dark mode.Can you share your think？ On Sep 26, 2024, at 10:42, FoXiYu ***@***.***> wrote:﻿
什么时候能出黑暗模式
或者自定义主题
随便问一下 选项里面的中文翻译好像没有 翻译这块是怎么提供的

**Reply by dail8859** [reply]
I also understand a lot of people like dark mode. It is a low priority feature that I'm only willing to implement if the necessary changes are easy to manage. To me there are other more important areas worth improving first.

**Reply by foxiyu** [reply]
It depends on personal usage habits. I prefer dark mode
